easybuild/easyblocks/generic/pythonbundle.py

```diff
@@ -120,6 +121,12 @@ def prepare_step(self, *args, **kwargs):
             orig_exts_filter = EXTS_FILTER_PYTHON_PACKAGES
             self.cfg['exts_filter'] = (orig_exts_filter[0].replace('python', python_cmd), orig_exts_filter[1])
 
+    def extensions_step(self):
```

Comment:
@Flamefire Please pass down arguments too:

```python
def extensions_step(self, *args, **kwargs):
    ...
    super(PythonBundle, self).extensions_step(*args, **kwargs)

Reply:
Did it this way because the other easyblocks do it too. Anyway, changed.

Reply:
Since this is a (commonly used) generic easyblock, it's worth being a bit more careful here.

The other easyblock should use this same approach, but the impact of not doing so is more likely to be non-existent there...